servicetalk-concurrent-api/src/main/java/io/servicetalk/concurrent/api/DefaultAsyncContextProvider.java

```diff
@@ -379,4 +378,15 @@ private static ContextMap exchangeContext(ContextMap contextMap) {
     private static ContextMap newContextMap() {
         return new CopyOnWriteContextMap();
     }
+
+    @SuppressWarnings("PMD.SystemPrintln")
+    private static boolean isNoDebugLogging() {
+        try {
+            return !LOGGER.isDebugEnabled();
+        } catch (Exception ex) {
+            // Logger isn't initialized, so we have to send it to a console.
+            System.err.println("Couldn't evaluate debug logging: defaulting to false. Cause:\n" + ex);
+            return true;
```

Comment:
What a play of words: defaulting to false, return true 😃 
Consider something like: 
```java
System.err.println("Could not evaluate logging level, considering debug level is disabled by default. Cause:" + System.lineSeparator() + ex);
ex.printStackTrace();
```